src/main/java/org/broadinstitute/hellbender/tools/copynumber/allelic/alleliccount/AllelicCountReader.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount;
+
+import org.broadinstitute.hellbender.utils.Nucleotide;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Reads {@link AllelicCount} instances from a tab-separated table file.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+final class AllelicCountReader extends TableReader<AllelicCount> {
```

Comment:
I think this instead should be a method of `AllelicCountCollection` that returns a new `TableReader<AllelicCount>` object, since it's a relatively short class and only used there for now